compiler/optimizer/OMROptimizationManager.cpp

```diff
@@ -49,7 +49,7 @@ OMR::OptimizationManager::OptimizationManager(TR::Optimizer *o, OptimizationFact
         _numPassesCompleted(0),
         _optData(NULL),
         _optPolicy(NULL),
-        _enabled(false),
+        _enabled(!o->comp()->isDisabled(optNum)),
```

Comment:
@andrewcraik agree with most of that, but compile time verification? you're really worried about the compile time impact of the queries to check if an optimization is enabled/disabled?

If you're worried about compile time regressions for this change, then I think you should be concerned about compile time regressions for any change to any optimization, which seems a trifle overboard, wouldn't you agree?